backend-diagram-converter/core/src/main/java/org/camunda/community/converter/RetryTimeCycleConverter.java

```diff
@@ -1,25 +1,58 @@
 package org.camunda.community.converter;
 
+import java.time.Duration;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class RetryTimeCycleConverter {
+  private static final Logger LOG = LoggerFactory.getLogger(RetryTimeCycleConverter.class);
   private static final Pattern PATTERN = Pattern.compile("R(\\d*)/");
+  private static final Pattern DURATION_PATTERN = Pattern.compile("(P.*)");
 
-  public static int convert(String timecycle) {
+  public static List<String> convert(String timecycle) {
```

Comment:
Who knows, what the list contains? (except you as implementor).

The process engine implementation offers getters for all important parts...

Reply:
Added an explanation